.releaserc

```diff
@@ -0,0 +1,25 @@
+{
+	"branches": ["version-13"],
+	"plugins": [
+		"@semantic-release/commit-analyzer",
+		"@semantic-release/release-notes-generator",
+		[
+			"@semantic-release/exec", {
+				"prepareCmd": 'sed -ir "s/[0-9]*\.[0-9]*\.[0-9]*/${nextRelease.version}/" erpnext/__init__.py'
```

Comment:
IIRC this one gladly bumps it to the next version 😅 

We should block it in case of a major version bump.

Reply:
This it will do only in case any breaking changes are merged which ideally should not go in the `version-13` branch anymore. Will anyways see if I can add a safety check